tools/run_tests/run_tests.py

```diff
@@ -759,9 +759,13 @@ def _start_port_server(port_server_port):
     env = dict(os.environ)
     env['BUILD_ID'] = 'pleaseDontKillMeJenkins'
     if platform.system() == 'Windows':
+      # Working directory of port server needs to be outside of Jenkins
+      # workspace to prevent file lock issues.
```

Comment:
The dir is only created once we start a new port server which should only happen once we bump its version.
As far as I remember, we had some issue with the TEMP env variable not being set in our win jenkins environment.